pyproject.toml

```diff
@@ -28,7 +28,7 @@ scikit-image = "*"
 fuzzywuzzy = "*"
 PyWavelets = "1.3.0"
 networkx = "*"
-pymedphys = ">=0.35,<=0.37.1"
+pymedphys = ">=0.35,<=0.37.2"

```

Comment:
This will break.  0.37.2 and later had the pseudonymisation interface disabled due to it's susceptibility to rainbow table attack.  0.39 had that fixed by adding pepper to the hashing.

Reply:
I changed the version to 0.39.3, my python version is 3.10.11